src/Microsoft.AspNet.WebHooks.Receivers/WebHooks/WebHookReceiver.cs

```diff
@@ -55,37 +53,37 @@ protected WebHookReceiver()
         internal static async Task<T> ReadAsJsonAsync<T>(HttpRequestMessage request)
             where T : JToken
         {
-            HttpConfiguration config = request.GetConfiguration();
+            var config = request.GetConfiguration();
 
             // Check that there is a request body
             if (request.Content == null)
             {
-                string msg = ReceiverResources.Receiver_NoBody;
+                var msg = ReceiverResources.Receiver_NoBody;
```

Comment:
LMK if you want "msg" -> "message" and perhaps "config" -> "configuration" in this PR. If so, I'll send out a separate PR to make those changes globally.

Reply:
Separate is fine